plane/source/docs/arming-throttle.rst

```diff
@@ -19,79 +19,75 @@ propeller.
    there is always a slight possibility that signal could unintentionally
    reach the motor. Even though this is unlikely (and made even less likely
    by safety features such as this) it only takes one time to chew up a
-   finger or hand!
-
+   finger or hand!
+
 Parameters governing throttle arming are introduced in the
-:ref:`ARMING <ARMING_CHECK>` section of the
+:ref:`ARMING <ARMING-PARAMETERS>` section of the

```

Comment:
This link would be broken. If you run the docs build it will download the parameters.rst file and you can see that the anchor for this section is defined as below:

```
.. _parameters_ARMING_:
```

That means your link actually has to look like this (ie dump the preceding underscore):
```
:ref:`ARMING <parameters_ARMING_>`
```